android/app/src/main/assets/config.json

```diff
@@ -2,6 +2,6 @@
 
   "email": "dev@fossasia.org",
   "app-name": "Open Event",
-  "api-link": "https://open-event-api-dev.herokuapp.com/v1/events/173",
+  "api-link": "https://open-event-api-dev.herokuapp.com/v1/events/1",
```

Comment:
Why changed?

Reply:
event 173 was giving error 404 so i checked for the active events using postman and set it to 1 just for time being

Reply:
There is no event in new API for download, but don't change this